content/04.results.md

```diff
@@ -25,6 +25,10 @@ Figure {@fig:mutational-landscape-overview}B-C show concordance of these samples
 ### Recurrence and co-occurrence of mutations
 
 This section will discuss the genes and regions that are repeatedly mutated within and between cancer types.
-The occurrence of mutations in affecting particular genes, separated by cancer,  type is shown in Figure {@fig:occurrence-plots}A, with significant co-occurrence across all types and within types with sufficient sample sizes illustrated in Figure {@fig:occurrence-plots}B.
+The occurrence of mutations in affecting particular genes, separated by tumor type is shown in Figure {@fig:occurrence-plots}A, with significant co-occurrence across all types illustrated in Figure {@fig:occurrence-plots}B.
 
-![Occurrence and co-occurrence of mutations](images/figure_sketches/occurrence-fig-sketch.jpg){#fig:occurrence-plots}
+![Occurrence and co-occurrence of nonsynonymous mutations for the 50 most commonly mutated genes across all tumor types.
+A) Counts of nonsynonymous mutations, colored by tumor type.
```

Comment:
We likely want to take this out in the future, but as we are writing this up I think it would be helpful to state which column you used for `tumor type`.

Reply:
Done.